llvmlite/tests/test_refprune.py

```diff
@@ -495,6 +496,28 @@ def test_fanout_raise_4(self):
         mod, stats = self.check(self.fanout_raise_4)
         self.assertEqual(stats.fanout_raise, 0)
 
+    fanout_raise_5 = r"""
+define i32 @main(i8* %ptr, i1 %cond) {
+bb_A:
+    call void @NRT_incref(i8* %ptr)
+    br i1 %cond, label %bb_B, label %bb_C
+bb_B:
+    call void @NRT_decref(i8* %ptr)
+    br label %common.ret
+bb_C:
+    br label %common.ret       ; pretend we throw an exception
+common.ret:
+    %common.ret.op = phi i32 [ 0, %bb_B ], [ 1, %bb_C ]
+    ret i32 %common.ret.op
+}
+
+!0 = !{i32 1}
```

Comment:
unused metadata

Reply:
Now that the test changed. The metadata should not be removed in https://github.com/numba/llvmlite/pull/830/commits/9523fadecfc638a1b6abf0494ff45c2aea17757d